app/src/main/java/org/mozilla/fenix/library/recentlyclosed/RecentlyClosedFragment.kt

```diff
@@ -30,33 +32,63 @@ import org.mozilla.fenix.components.FenixSnackbar
 import org.mozilla.fenix.components.StoreProvider
 import org.mozilla.fenix.ext.getRootView
 import org.mozilla.fenix.ext.requireComponents
+import org.mozilla.fenix.ext.setTextColor
 import org.mozilla.fenix.ext.showToolbar
 import org.mozilla.fenix.library.LibraryPageFragment
 
 @Suppress("TooManyFunctions")
-class RecentlyClosedFragment : LibraryPageFragment<RecoverableTab>() {
+class RecentlyClosedFragment : LibraryPageFragment<RecoverableTab>(), UserInteractionHandler {
     private lateinit var recentlyClosedFragmentStore: RecentlyClosedFragmentStore
     private var _recentlyClosedFragmentView: RecentlyClosedFragmentView? = null
     protected val recentlyClosedFragmentView: RecentlyClosedFragmentView
         get() = _recentlyClosedFragmentView!!
 
     private lateinit var recentlyClosedInteractor: RecentlyClosedFragmentInteractor
+    private lateinit var recentlyClosedController: RecentlyClosedController
 
     override fun onResume() {
         super.onResume()
         showToolbar(getString(R.string.library_recently_closed_tabs))
     }
 
     override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
-        inflater.inflate(R.menu.library_menu, menu)
+        if (recentlyClosedFragmentStore.state.selectedTabs.isNotEmpty()) {
+            inflater.inflate(R.menu.history_select_multi, menu)
+            menu.findItem(R.id.delete_history_multi_select)?.title =
+                SpannableString(getString(R.string.bookmark_menu_delete_button)).apply {
```

Comment:
We should introduce a separate string for this instead of re-using the bookmarks one.